test/e2e-test/testdata/app/trait_config.yaml

```diff
@@ -0,0 +1,49 @@
+# Code generated by KubeVela templates. DO NOT EDIT.
+apiVersion: core.oam.dev/v1beta1
+kind: TraitDefinition
+metadata:
+  name: config
+spec:
+  status:
+    customStatus: |-
+      if context.outputs.config.kind == "Secret" {
+        message: "secret:" + context.outputs.config.metadata.name
+      }
+      if context.outputs.config.kind == "ConfigMap" {
+        message: "configMap:" + context.outputs.config.metadata.name
+      }
+  appliesToWorkloads:
+    - deployments.apps
+  podDisruptive: true
+  schematic:
+    cue:
+      template: |-
+        outputs: config: {
```

Comment:
```suggestion
        outputs: parameter.configname: {
```

Reply:
got it

Reply:
Users maybe need use variable of `parameter` to get the resource in `context.outputs`, so we add `parameter` context for  `spec.status.customStatus`, users can get variable in `parameter` by `parameter.configname `